deploy/helm/Chart.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: v1
+appVersion: 3.1.0
+description: nfs-subdir-external-provisioner is an automatic provisioner that used your *already configured* NFS server, automatically creating Persistent Volumes.
+name: nfs-subdir-external-provisioner
+home: https://github.com/kubernetes-sigs/nfs-subdir-external-provisioner
+version: 3.0.0
+kubeVersion: ">=1.9.0 <1.20.0"
```

Comment:
is max verison still needed given latest update in https://github.com/kubernetes-sigs/nfs-subdir-external-provisioner/issues/25 ?

Reply:
answering myself: yes, because of course the change referenced in #25 has not even been released yet!

Reply:
exactly 👍